frontend/app-development/layout/SettingsModalButton/SettingsModal/components/Tabs/AccessControlTab/AccessControlTab.tsx

```diff
@@ -92,13 +71,13 @@ export const AccessControlTab = ({ org, app }: AccessControlTabProps): ReactNode
         );
       }
       case 'success': {
+        const currentPartyTypesAllowed = appMetadata?.partyTypesAllowed ?? initialPartyTypes;
         return (
           <Checkbox.Group
             legend={t('settings_modal.access_control_tab_checkbox_legend')}
             size='small'
-            onChange={handleChange}
-            onBlur={handleSavePartyTypes}
-            value={Object.keys(partyTypesAllowedMap).filter((key) => partyTypesAllowed[key])}
+            onChange={(newValues: string[]) => handleChange(newValues, currentPartyTypesAllowed)}
+            value={Object.keys(partyTypesAllowedMap).filter((key) => currentPartyTypesAllowed[key])}
```

Comment:
Here we do `Object.keys` of `partyTypesAllowedMap`, but on line 46 we do `Object.keys` of `currentPartyTypesAllowed`. 

If I understand this correctly they both will always have the same result, since they both are `type PartyTypesAllowedOption = 'bankruptcyEstate' | 'organisation' | 'person' | 'subUnit'; `

I think we could create a variable on root of `AccessControlTab.tsx`, `const allowedPartyTypes = Object.keys(partyTypesAllowed)`, then use the `allowedPartyTypes` to loop through the keys? 🤔

Reply:
Yes, I'll fix it 😄